### ISSUE TITLE
404 not found after installation

### ISSUE THREAD

**Issue by schier1987** [BookStackApp/BookStack#1479]
Hi,

I just installed bookstack following the instructions @ https://www.bookstackapp.com/docs/admin/installation/ (Manual Installation)

Everything went fine, no errors or anything, but when I open the URL, I get instantly redirected to /login, throwing "404 Not found".

As it sounded similar, I already tried chaning database collation to utf8mb4_unicode_ci as described here https://github.com/BookStackApp/BookStack/issues/1050

I cloned the newest BookStack-Version from git, my server runs PHP 7.0.33-0ubuntu0.16.04.4

I couldn't find anything in the server logs...

Do you have any idea?

thanks and best regards

**Comment by ssddanbrown** [comment]
Hi @schier1987,
What kind of 404 page do you see? A fairly blank page or a BookStack (Blue-looking) page?

Additionally, what web server and operating system are you running?

**Comment by schier1987** [comment]
Hi @ssddanbrown 

Thanks for your answer.

Fairly blank, not BookStack related.

I am running apache on Ubuntu 16.04, and I set the .htaccess as explained in the docs (respectively the installation did it by itself)

**Comment by GonzaChocan** [comment]
dsaasd

**Comment by chefarbeiter** [comment]
HI @schier1987 ,

are you using nginx as web server?

I had the same issue after a fresh install of BookStack on Ubuntu. Error page 404 from nginx, no message in any log file. Access to static files in the public folder (e. g. user_avatar.png) via web server was successfull. Then I replaced the original index.php by a dummy one -> same error -> BookStack wasn't the problem here.

I checked my nginx-config for the site many times and ... finally found the problem :smiley:

I had the location-block for php-files outside of the root-location-block:
```nginx
location / {
   root /var/www/docs/bookstack/public;
   try_files $uri $uri/ /index.php?$query_string;
}

location ~ [^/]\.php(/|$) {
  [...]
  fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
  [...]
}
```

But the php-location must be placed **inside** the root-location:
```nginx
location / {
   root /var/www/docs/bookstack/public;
   try_files $uri $uri/ /index.php?$query_string;

   location ~ [^/]\.php(/|$) {
      [...]
      fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
      [...]
   }
}
```

Reloaded nginx and BookStack is working! :smiley:
